### PR TITLE
[full-ci] Make ui small again

### PR DESCRIPTION
**Pull request by lookacat**
## Description
See https://github.com/owncloud/web/issues/7333
Needs https://github.com/owncloud/owncloud-design-system/pull/2270

## Related Issue

- Fixes https://github.com/owncloud/web/issues/7333

## Screenshots (if appropriate):
Screenshots are done on a pretty large screen (both display 10 files)
Before
image
After
image
Dropbox as reference
image

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: packages/web-runtime/src/App.vue
Comment (kulmann) [inline packages/web-runtime/src/App.vue:206]: Please use css variables from the design system for font sizes

**Review by kulmann — COMMENTED** [review]
Which ticket are you working on here? 🤔 please link it in the PR description

File: packages/web-runtime/src/App.vue
Comment (lookacat) [reply packages/web-runtime/src/App.vue:206]: yes will change it, was just an test

**Comment by tbsbdr** [comment]
screenshots current state for comparison before - after
Screenshot 000176@2x
Screenshot 000179@2x
Screenshot 000178@2x
Screenshot 000177@2x

**Review by tbsbdr — APPROVED** [review]

**Review by kulmann — CHANGES_REQUESTED** [review]
needs another ODS update before merge. please don't merge, yet.

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingAutocompletion</strong> https://drone.owncloud.com/owncloud/web/27569/63/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUISharingAutocompletion-shareAutocompletionSpecialChars_feature-L80.png
  


</p></details>
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Review by kulmann — APPROVED** [review]
Good now 👍

**Pull request merged**
